books/views/external_books_view.py

```diff
@@ -0,0 +1,17 @@
+from requests import get
+from rest_framework import generics, status
+
+from books.helpers.rest_helpers import make_formatted_response
+from books.serializers import ExternalBooksSerializer
+
+
+class ListIceAndFireBooksView(generics.ListAPIView):
+
+    def get_queryset(self, request):
+        res = get("https://www.anapioficeandfire.com/api/books", request.query_params)
```

Comment:
@xhahid43eb This isn't a proper use of get_queryset, query set is meant to pull something out of internal database models. You can rename this function.

Reply:
got it.